### ISSUE TITLE
Appearance Themes- Screen reader announces “No file chosen button” instead of “Choose File”

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1133]
"Description: On the Configure Theme page in Appearance Themes section, file input fields including Background Image, Logo File, and Header Image  visually show “Choose File”. It was found that sometimes the 'Choose File button' is not announced or is announced incorrectly as follows:

Windows Chrome (NVDA+JAWS): When the button receives focus, it announces “No file chosen button” and does not announce the 'Choose File' button. This can be confusing to users as part of the component is not announced.

On MacOS Safari (VoiceOver): When the button receives focus, the button is announced. However, it is announced as 'no file selected Background Image, file upload button'. The 'File Upload' button does not match the visible label of 'Choose File'.

Note: This does not occur on Windows Firefox browser. The 'Choose File' button is instead a 'Browse' button and is announced alongside the 'No file selected' text. 

Expected Results: The screen reader should announce “Choose File” (matching the visual label) rather than the default browser text. Users should clearly understand that the control is for selecting a file.

Actual Results: The screen reader announces “No file chosen button”.
This exposes the internal default state of the input instead of the intended label. Users may be confused about the control’s purpose."

4.1.2 Name, Role, Value; Level A; WCAG 2.0
2.5.3 Label in Name; Level A; WCAG 2.1
